safe_transaction_service/history/helpers.py

```diff
@@ -56,12 +56,18 @@ def calculate_all_possible_hashes(cls, delegate: ChecksumAddress) -> List[bytes]
 
 def is_valid_unique_transfer_id(unique_transfer_id: str) -> bool:
     """
-    Check if transfer_id starts with 'e' or 'i' followed by keccak256 and appended by optional digits separated by commas
+    Check if transfer_id starts with 'e' or 'i' followed by keccak256 and appended by digits or digits separated by commas
```

Comment:
Ended instead of appended I think it's more clear

Reply:
Sounds good